recipes/cni-plugins/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cni-plugins"%}
+{% set version = "0.7.0"%}
+{% set sha256 = "8775ab7d8172b040091d7bbbe2b789ce6336bed9b5921ee05de1f8b2a6d16f6c" %}
+{% set cni_version = "0.6.0"%}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/containernetworking/plugins/archive/v{{ version }}.tar.gz  # [not win]
+  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
+  sha256: {{ sha256 }}  # [not win]
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - cni {{ cni_version }}
+    - go 1.8.*  # [not win]
+    - toolchain  # [not win]
+  run:
+    - cni {{ cni_version }}
+
+test:
+  commands:
+    - test -f $CNI_PATH/bridge
```

Comment:
The plugins must be executed by cnitool, which in turn requires root access to finish correctly.